### PR TITLE
Added open_sourcing_fork_cms_modules_and_themes.md

### PR DESCRIPTION
**Pull request by jeroendesloovere**

### CODE REVIEW

File: contribute/open_sourcing_fork_cms_modules_and_themes.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Open Sourcen Fork CMS Modules and Themes
```
Comment (WouterSioen) [inline contribute/open_sourcing_fork_cms_modules_and_themes.md]: this should probably be "open sourcing"

**Comment by WouterSioen** [comment]
It could maybe be a good idea to also add a link to this repo? https://github.com/guytarded/fork-cms-extensions

Persons open sourcing modules could use this for two things: checking if the module they want to open source already exists and adding their own module to it.

**Comment by carakas** [comment]
@jeroendesloovere ^

**Comment by jeroendesloovere** [comment]
@carakas done

**Pull request merged**